TURN reports failed to handle Refresh-request

**Describe the bug**
When running in production, the logs would print lines like:

```
turn ERROR: 2021/10/20 18:55:24 error when handling datagram: failed to handle Allocate-request from 212.225.225.45:21336: relay already allocated for 5-TUPLE
turn ERROR: 2021/10/20 19:21:13 error when handling datagram: failed to handle Refresh-request from 212.225.225.45:12676: no allocation found 212.225.225.45:12676:0.0.0.0:443
```

**Server**
- Version: 0.13.5

**Client**
- SDK: Web

**To Reproduce**
It's not clear how to reproduce this. Related tickets mention FireFox 91 on Android. https://github.com/pion/turn/issues/206

Having the same issue:
TEST results:
Connecting to signal connection via WebSocket

Connected to server, version 0.15.3.

Establishing WebRTC connection

Can connect via TURN

Can publish audio

published 62 audio packets

Can publish video

ERROR: Could not get RTCStats

Resuming connection after interruption

------ LOGS

livekit_1  | 2022-02-09T05:52:00.675Z   ERROR   livekit.turn    v2@v2.0.6/server.go:184 error when handling datagram: failed to handle Allocate-request from 41.77.144.98:51596: relay already allocated for 5-TUPLE

--- a/go.mod
+++ b/go.mod
@@ -27,12 +27,13 @@ require (
 	github.com/pion/sdp/v3 v3.0.4
 	github.com/pion/stun v0.3.5
 	github.com/pion/transport v0.13.0
-	github.com/pion/turn/v2 v2.0.6
+	github.com/pion/turn/v2 v2.0.8
 	github.com/pion/webrtc/v3 v3.1.23
 	github.com/pkg/errors v0.9.1
 	github.com/prometheus/client_golang v1.11.0
 	github.com/rs/cors v1.8.2
 	github.com/rs/zerolog v1.26.0
+	github.com/sebest/xff v0.0.0-20210106013422-671bd2870b3a
 	github.com/stretchr/testify v1.7.0
 	github.com/thoas/go-funk v0.8.0
 	github.com/twitchtv/twirp v8.1.0+incompatible
@@ -72,7 +73,6 @@ require (
 	github.com/prometheus/common v0.26.0 // indirect
 	github.com/prometheus/procfs v0.6.0 // indirect
 	github.com/russross/blackfriday/v2 v2.1.0 // indirect
-	github.com/sebest/xff v0.0.0-20210106013422-671bd2870b3a // indirect
 	go.uber.org/multierr v1.6.0 // indirect
 	golang.org/x/crypto v0.0.0-20220208050332-20e1d8d225ab // indirect
 	golang.org/x/mod v0.5.1 // indirect

